hatchery/nextflow.go

```diff
@@ -1474,3 +1558,23 @@ workDir = '%s'`,
 
 	return configContents, nil
 }
+
+// function to generate user data
+func generateUserData(userName string) (string, error) {
+	// TODO: read repo from config
+	approvedRepo := "143731057154.dkr.ecr.us-east-1.amazonaws.com/nextflow-approved"
+
+	// TODO: read region from config
+	userData := base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf(`MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
+
+--==MYBOUNDARY==
```

Comment:
what does MYBOUNDARY do?

Reply:
It's the marker for the start and end for the multiline string. Kind of similar to `cat << EOF`